Airports/EGBS/Positions.txt

```diff
@@ -0,0 +1 @@
+EGBS_I_TWR:Shobdon Information:118.150:BST:I:EGBS:TWR:-:-:::N052.14.30.000:W002.52.53.000
```

Comment:
```suggestion
EGBS_I_TWR:Shobdon Information:118.150:BSI:I:EGBS:TWR:-:-:::N052.14.30.000:W002.52.53.000
```

Reply:
Are we standardising these at some point? I've noticed quite a bit of variation in the Sector File with regards to FISO positions. Some are xxT and some are xxI. Happy to go with I as I think it makes more sense because it's not really a TWR. :)

Reply:
Leaving this as unresolved pending clarification so I don't teach others wrong in the future.